pom.xml

```diff
@@ -4219,6 +4220,246 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
```

Comment:
The rebase seems to have gone a bit wrong here. These should no longer be in the parent pom and they should be in the `ee-feature-pack/pom.xml`.